### ISSUE TITLE
A11y_Manage.IIS.Net_Installation Popup_Color Contrast: In high contrast black mode, unable to see "I agree to the license terms and condition" beside the checkbox 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/IIS.WebManager#215]
**MAS ID Violated:**
WCAG 1.4.3

**Repro Steps**:
1. Hit the Applicaton URL 
2. Tab till "Download Microsoft IIS Administration" button and press enter and run "IISAdministrationSetup.exe" file
3. Turn on high contrast black mode
4. Verify in high contrast black mode, user is able to see "I agree to the license terms and condition" beside the checkbox or not

**Actual Result:**
 In high contrast black mode, unable to see "I agree to the license terms and condition" beside the checkbox 

**Expected Result:**
 In high contrast black mode, user should able to see "I agree to the license terms and condition" beside the checkbox 

**User Impact:**
Visually impaired users finds difficult in viewing controls and content while working with this application

**Attachment:**
![bug_215 _installation popup_high contrast black](https://user-images.githubusercontent.com/39204909/46394936-440a6d80-c708-11e8-94ca-d94dbdbf74d4.png)

**Comment by jhkimnew** [comment]
Duplicated https://github.com/Microsoft/IIS.Administration/issues/221

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ColorContrast; #MAS1.4.3; #C+E-H; #.NETWebsites; #Edge

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
